source/NVDAObjects/window/winword.py

```diff
@@ -219,6 +219,28 @@
 	wdThemeColorText2:msoThemeDark2,
 }
 
+# map index from wdColorIndex to decimal values in wdColor
+highlightColors = {

```

Comment:
Could you document the relevant enums here?

https://learn.microsoft.com/en-us/office/vba/api/word.wdcolorindex
https://learn.microsoft.com/en-us/office/vba/api/word.wdcolor

It would be even better to emulate these enums with python `IntEnum` and then making the mapping as follows:

```suggestion
class WinWordColorIndex(IntEnum):
	wdColorBlack = 1
	....
	
class WinWordColor(IntEnum):
	wdColorBlack = 0
	....

_highlightColors: Dict[WinWordColorIndex, WinWordColor) = {
```

Reply:
I'm not sure to have understood your proposal. After making enums, do you say to define _highlightColors as `{i.value: WinWordColor[i.name].value for i in WinWordColorIndex}`?

Reply:
yes exactly.

It also may be better named `_colorIndexToColor`

Reply:
Ok, I'm proceeding. Just for clarity, to avoid reverting commits: I'll build WinWordColorIndex enum renaming i.e. wdBlack (as it would be in original enum) to wdColorBlack, and excluding reference-only values (wdByAuthor, wdAuto, wdNoHighlight). Then, in WinWordColor enum, I'll include just wdColor* useful for _colorIndexToColor mapping (16 against around 60). Right? Or do you prefer if I document all colors? (sorry for questions, this is my first PR)

Reply:
Yes, no need to map values that are not used in NVDA.

Your proposal makes sense.
It also may be worth dropping the `wdColor` prefix on the enum items, if you can do it consistently for all items.